easybuild/easyconfigs/a/Autotools/Autotools-20150119-intel-2015a.eb

```diff
@@ -0,0 +1,18 @@
+easyblock = 'Bundle'
+
+name = 'Autotools'
+version = '20150119' # date of the most recent change
+
+homepage = 'http://autotools.io'
+description = """This bundle collect the standard GNU build tools: Autoconf, Automake and libtool"""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
```

Comment:
Having dependencies across toolchains doesn't inherently work in a HMNS. For example, intel depends on GCC 4.X but you don't want to have that GCC extend the path since this could cause headaches if you install software there (module files with different toolchains have exactly the same name, unlike in a flat scheme). If you really want the lowest common denominator you should probably use `dummy`